include/openthread/platform/logging.h

```diff
@@ -57,18 +55,41 @@ extern "C" {
 /**
  * Log levels.
  *
- * Implementation note: These exist as defines (not enums) so that
- * embedded code can remove code at compile time via if/else/endif
+ * Implementation note: Log Levels are defines so that embedded implementations can eliminate code at compile time via
+ * #if/#else/#endif.
+ *
  */
-
 #define OT_LOG_LEVEL_NONE 0 ///< None
 #define OT_LOG_LEVEL_CRIT 1 ///< Critical
 #define OT_LOG_LEVEL_WARN 2 ///< Warning
 #define OT_LOG_LEVEL_NOTE 3 ///< Note
 #define OT_LOG_LEVEL_INFO 4 ///< Info
 #define OT_LOG_LEVEL_DEBG 5 ///< Debug
 
-/* note: The enum otLogRegion is found in 'openthread/types.h' */
+typedef uint8_t otLogLevel;
```

Comment:
My script is throwing an error that otLogLevel is not documented, probably because now the defines are between it and the comments.  Currently it's been pulling in this:

https://openthread.io/reference/group/api-types.html#group__api-types_1ga541df22d7943d447bc99f16f517f6abd

Reply:
Thanks. I made some updates to this. Let me know if the updates help.

Reply:
Yes that's much better.